molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/wagtailadmin/pages/listing/_page_title_explore.html

```diff
@@ -32,7 +32,14 @@
         <li><a href="{% url 'wagtailadmin_pages:copy' page.id %}" class="button button-small button-secondary">{% trans 'Copy' %}</a></li>
     {% endif %}
     {% if page_perms.can_delete and 'delete' not in hide_actions|default:'' %}
-        <li><a href="{% url 'wagtailadmin_pages:delete' page.id %}" class="button button-small button-secondary">{% trans 'Delete' %}</a></li>
+        {# BEGIN CUSTOM TEMPLATE LOGIC #}
```

Comment:
@nathanbegbie would you mind adding a comment around what you add here, so for future will be easier to see what we add to wagtail initial template